ferrocene/doc/requirements/src/self-test.rst

```diff
@@ -140,6 +124,16 @@ relative to the root of the installation,
 should exist the regular file named ``ld.lld``,
 which is the linker wrapper.
 
+Platforms that need a C compiler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+   The following checks apply to these Host platforms:
+
+   - :target:`x86_64-unknown-linux-gnu`
+   - :target:`aarch64-unknown-linux-gnu`
```

Comment:
I would like if we are able to integrate that information into `user-manual:targets/index` as well in the future. This information is present in https://github.com/ferrocene/ferrocene/blob/82ba226a05d24f646d270be57ff6e12f9f0173fa/ferrocene/tools/self-test/src/targets.rs#L12

Reply:
Emphasis on "in the future", not in this PR.